sed/config/default.yaml

```diff
@@ -31,6 +31,8 @@ dataframe:
   tof_binning: 1
   # binning factor used for the adc coordinate (2^(adc_binning-1))
   adc_binning: 1
+    # list of columns to apply jitter to
+  jitter_cols: ["@x_column", "@y_column", "@tof_column"]
 
```

Comment:
I would suggest addding here `jitter_amps: [0.5,0.5,0.5]`. this would be redundant here, but in the case of hextof would actually be `jitter_amps: [0.5,0.5,4]`

Reply:
My very strong preference would be to fix this in the flash loader, and have correct digital units on the flash dldTime, and leave amps out of the config... we still have this kwd access in case one needs it for debugging.

Reply:
I disagree though, as this is not general. 
There might be other cases in which a digial channel is not represented in units of the digital step.
We don't have an example here, but I had similar experiences with continuous values (voltage) measured through an ADC which had floating point step size. Was digital but represented as float.

And why such a strong preference? does this break something or make something hard to understand?

Reply:
I fixed this in the flash loader

Reply:
> And why such a strong preference? does this break something or make something hard to understand?

Having exactly the right jitter parameters is quite important if you want the digitzation artefacts to disappear, but no loss of resolution. That's why I would rather keep this away from the user and as the responsibility of the instrument scientist setting up the config file to check this carefully. We should add an example notebook for the influence of the jittering.

Reply:
Having said this, I does not provide a very good reason against having it as a config parameter, so I have added it now.